sample-code/src/orchestration.ts

```diff
@@ -39,6 +41,32 @@ export async function orchestrationChatCompletion(): Promise<OrchestrationRespon
   return result;
 }
 
+/**
+ * Ask GPT 3.5 through the orchestration service about SAP Cloud SDK with streaming.
```

Comment:
```suggestion
 * Ask GPT 4o through the orchestration service about SAP Cloud SDK with streaming.
```

Reply:
I changed the endpoint to use 3.5 turbo instead (since that's cheaper)